contracts/rewards/RewardsDistributor.sol

```diff
@@ -513,4 +515,24 @@ abstract contract RewardsDistributor is IRewardsDistributor {
   function getAssetDecimals(address asset) external view returns (uint8) {
     return _assets[asset].decimals;
   }
+
+  /// @inheritdoc IRewardsDistributor
+  function getEmissionManager() external view returns (address) {
+    return _emissionManager;
+  }
+
+  /// @inheritdoc IRewardsDistributor
+  function setEmissionManager(address emissionManager) external onlyEmissionManager {
+    _setEmissionManager(emissionManager);
+  }
+
+  /**
+   * @dev Update the EmissionManager admin to the provided address parameter
+   * @param emissionManager The address of the new emission manager
+   */
+  function _setEmissionManager(address emissionManager) internal {
+    require(emissionManager != address(0), 'EMISSION_MANAGER_NOT_ZERO');
+    _emissionManager = emissionManager;
+    emit EmissionManagerUpdated(emissionManager);
```

Comment:
Add the old emissionManager value to the event
`emit EmissionManagerUpdated(oldEmissionManager, newEmissionManager)`